### PR TITLE
[15.0][FIX] stock_owner_restriction: Wrong qty available computation for owner and product at the same time

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT50899

ping @carlosdauden @chienandalu @rousseldenis

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
Seems fair

**Comment by rousseldenis** [comment]
@sergio-teruel Should be great to add a little test for that (and maybe in README too?)

**Review by chienandalu — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]

File: stock_owner_restriction/models/product.py [hunk 1]
```diff
@@ -34,3 +34,11 @@ def _search_qty_available_new(
         return super(ProductProduct, new_self)._search_qty_available_new(
             operator, value, lot_id=lot_id, owner_id=owner_id, package_id=package_id
         )
+
+    @api.depends_context("force_restricted_owner_id")
+    def _compute_quantities(self):
+        # Add force_restricted_owner_id to depends_context to compute qty when this
+        # key change.
+        # For example, in a report you want to print all stock available from a product
+        # and and stock available from one owner
```
Comment (pedrobaeza) [inline stock_owner_restriction/models/product.py]:
```suggestion
        # and stock available from one owner
```

**Review by pedrobaeza — COMMENTED** [review]
@sergio-teruel are you adding the tests?

**Comment by sergio-teruel** [comment]
Pending to add tests..

**Comment by sergio-teruel** [comment]
Changes done!!

@rousseldenis I do not need a new test only remove the invalidate_cache in this test (https://github.com/OCA/stock-logistics-workflow/pull/1728/files#diff-5a28e7658e415aaa39f5bcfcbf1ed6888fd8acb55164767f5a554f9285a21b5aR69)

**Pull request merged**